Makefile

```diff
@@ -37,8 +38,10 @@ RELATED_IMAGE_OPENSCAP_TAG?=1.3.5
 # Image path to use. Set this if you want to use a specific path for building
 # or your e2e tests. This is overwritten if we bulid the image and push it to
 # the cluster or if we're on CI.
+DEFAULT_IMAGE_OPERATOR_PATH=quay.io/compliance-operator/$(APP_NAME):latest
+DEFAULT_IMAGE_OPENSCAP_PATH=quay.io/compliance-operator/$(RELATED_IMAGE_OPENSCAP_NAME):$(RELATED_IMAGE_OPENSCAP_TAG)
```

Comment:
Do I see correctly that this variable definition is twice in the Makefile?

Reply:
This is not the same variable: one for operator and one for openscap

Reply:
Oups I see, DEFAULT_IMAGE_OPENSCAP_PATH declaration at line 2 was an extra paste. Need to be removed